# Regression: Football Manager 2022 Player Data

Football Manager 2022 (officially abbreviated as FM22) is a 2021 football-management simulation video game developed by Sports Interactive and published by Sega. It was released worldwide for Microsoft Windows and macOS on 8 November 2021. Football Manager 2022 Xbox Edition was released on Xbox Game Pass, and Football Manager 2022 Mobile for iOS and Android on the same day. The streamlined version of the game titled Football Manager 2022 Touch was released exclusively for Nintendo Switch on 8 November 2021 {cite}`wikipedia_2022`.

## About the dataset

The data "[Football Manager 2022 Player Data](https://www.kaggle.com/code/gabrielabilleira/predicting-current-ability-for-players-in-fm2022)" is sourced from kaggle.

### Data Desciption

Player data from the game Football Manager 2022 that contains statistics from player performance in matches and transfer value. The data is obtained directly from the game and it is aimed mostly to try to predict the Current Ability of the players.  
Current Ability (CA) measures the player's ability for a specific position. It has a direct relation with the players attributes (at least the ones that are important for said position). This measure is not directly observable in the game. Scouts can fail depending on their ability and they may not be able to determine the players ability with high accuracy since they do that manually. Moreover, calculating the CA for every player would be very slow. Hence it is required to automate the process.

In [1]:
# Read the data
df <- read.csv('./../data/regression_data/input/player_data.csv')

In [2]:
# Get the dimensions of data
dim(df)

[1] 1122   45

There are 1122 observations and 44 features in the data with 1 target variable.

### Feature Desciption

The data used for this purpose comes from each player performance, which is easier to obtain and is usually more abundant.

In [4]:
colnames(df)

[1] "X"                "Name"             "Apps"             "Mins"            
 [5] "Mins.Gm"          "Height"           "Weight"           "Age"             
 [9] "Av.Rat"           "Gls"              "Gls.90"           "Shot.."          
[13] "Pen.R"            "xG"               "Ch.C.90"          "Asts.90"         
[17] "K.Ps.90"          "Pas.."            "Cr.C.A"           "Drb.90"          
[21] "Distance"         "Hdr.."            "K.Tck"            "Fls"             
[25] "Int.90"           "Clear"            "Con.90"           "xSv.."           
[29] "Sv.."             "Svh"              "Svt"              "Svp"             
[33] "Pens.Saved.Ratio" "PoM"              "Aer.A.90"         "Off"             
[37] "Based"            "Tck.R"            "CA"               "Saves"           
[41] "Saves.xSv."       "Gls.xG"           "Dist.Mins"        "Transfer.Value"  
[45] "Value"

|Sr. No.|Features|Description|
|--|:--|:--|
|1|X|Index number of the observation|
|2|Name|Name of the player|
|3|Apps|Number of appearances|
|4|Mins|Total Minutes played|
|5|Mins/Gm|Minutes played per game|